Java/Fibonacci Search/FibonacciSearch.java

```diff
@@ -1,5 +1,20 @@
 package Search;
 
+/* Copyright 2018 Aman Mehara
```

Comment:
Please write your own name.
Also change the year to 2020.
😃

Reply:
Ahhh ... My bad, I misunderstood the contribution guidelines.
Thanks for Spotting, Correction has been done.